apps/etterlatte-saksbehandling-ui/server/src/config/config.ts

```diff
@@ -106,6 +108,9 @@ const API_CONFIG_FROM_ENV = () => {
 
 export const ApiConfig = hentApiConfigFraEnv()
 
+const appVersion = crypto.randomUUID()
```

Comment:
Tror denne kommer til å støte på problemer hvis (når?) du treffer forskjellige pod'er -- er det mulig å bygge inn noe form for commit-hash-basert appversjon i stedet?

Reply:
Godt poeng! Jeg fant ingen defaulte her https://docs.nais.io/deployment/?h=default+env#default-environment-variables
Men vi kan kanskje bare sette en selv via env til frackend ja.

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/2088/commits/de2459072c63f736cd3a374816bcefe4cd7b1ae5